### PR TITLE
Holdout Sizegun

### PR DESCRIPTION
**Pull request by Mewchild**
Adds the holdout sizegun, a 2 shot version of the weapon that can be selected in the utility section of the loadout panel. Costs 4 points. 

https://forum.vore-station.net/viewtopic.php?f=26&p=8922#p8921

### CODE REVIEW

**Comment by Cameron653** [comment]
I assume it's being left so it auto recharges intentionally?

**Comment by Mewchild** [comment]
Default Sizeguns don't recharge automatically, do they? You have to put them in a recharger, from my admittedly limited experience

File: code/modules/vore/resizing/sizegun_vr.dm [hunk 1]
```diff
@@ -82,3 +82,9 @@
 			H.updateicon()
 		else
 			return 1
+
+//Two shot version for loadouts
+/obj/item/weapon/gun/energy/sizegun/small
+	name = "holdout size gun" //I have no idea why this was called shrink ray when this increased and decreased size.
```
Comment (KasparoVy) [inline code/modules/vore/resizing/sizegun_vr.dm]: Technically still half right. 😎

**Comment by Arokha** [comment]
Removes usefulness of science and maint loot. No from me.

**Pull request closed without merging**